### PR TITLE
feat(community): update bio @boennemann

### PR DESCRIPTION
**Pull request by boennemann**
Goes together with https://github.com/hoodiehq/hoodie-css/pull/49

Happy for any other suggestions, but the current one was just _painfully_ outdated ;)

### CODE REVIEW

File: community/index.html [hunk 1]
```diff
@@ -288,7 +288,7 @@ <h1 class="h3">
                 <a name="stephan-boennemann" class="jump-mark">Stephan Bönnemann</a>
             </h1>
             <p>
-                Studies MultiMediaTechnology and loves playing frisbee. Coffee Geek, soccer fan and Co-Organiser of <a href="http://conc.at/" target="_blank">.concat()</a>.
+                Freelance JavaScript developer, inventor of <a href="https://greenkeeper.io" target="_blank">Greenkeeper</a> and maintainer of <a href="https://github.com/semantic-release/semantic-release" target="_blank">semantic-release</a>. Enjoys cheesecake, coffee, cocktails and community.
```
Comment (NickColley) [inline community/index.html:291]: What kind of cheesecake do you recommend?

Comment (boennemann) [inline community/index.html:291]: The coffeeshop in my house sells Five Elephant Cheesecake which is the best I've ever had. You should totally try it if you're in Berlin. And I hope I'll manage to get their new variant in my face soon: https://www.instagram.com/p/BDSckRBKYN4/

**Comment by gr2m** [comment]
LGTM :)

**Comment by boennemann** [comment]
LGTM then :)

**Pull request merged**